paddle/fluid/operators/mkldnn/fc_mkldnn_op.cc

```diff
@@ -327,6 +409,20 @@ class FCMKLDNNHandler
 
   std::shared_ptr<dnnl::memory> AcquireCustomDstMemory(
       const ExecutionContext& ctx, phi::DenseTensor* out) {
+    if (ctx.HasAttr("fuse_residual_connection") &&
+        ctx.Attr<bool>("fuse_residual_connection")) {
+      auto* residual_param = ctx.Input<phi::DenseTensor>("ResidualData");
+      PADDLE_ENFORCE_EQ(
+          out->dims(),
+          residual_param->dims(),
+          phi::errors::InvalidArgument(
+              "Output and elementwise parameter need to have the "
```

Comment:
after using post_binary_add, do we still need to force residual data and dst to have same dims? I just think binary-add should support broadcast.